specs/fromPairs.test.ts

```diff
@@ -17,7 +17,7 @@ describe('fromPairs', () => {
         ['a', 1],
         ['b', 2],
         ['c', 3],
-        ['d', 4],
+        ['d', 4]
```

Comment:
There should be a comma at the end. 

When the list(be it is an object, array, argument list in function) is multiline, there will be a comma at the end of each line.

If the list is a single line, the comma after each element except the last element;
 space after each comma in such list.

Please update on other places too.